inc/hero-text-widget.php

```diff
@@ -0,0 +1,253 @@
+
+		parent::__construct( 'primer-hero-text', esc_html__( 'Hero Text', 'primer' ), $widget_options );
+
+		add_action( 'admin_enqueue_scripts',              array( $this, 'enqueue_scripts' ) );
+		add_action( 'customize_controls_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+
+	}
+
+	/**
+	 * Display the widget on the front-end.
+	 *
+	 * @since NEXT
+	 *
+	 * @param array $args     Display arguments including `before_title`, `after_title`, `before_widget`, and `after_widget`.
+	 * @param array $instance The settings for the particular instance of the widget.
+	 */
+	public function widget( $args, $instance ) {
+
+		/**
+		 * Filter the widget title.
+		 *
+		 * @link  https://developer.wordpress.org/reference/hooks/widget_title/
+		 * @since NEXT
+		 *
+		 * @param array  $instance An array of the widget's settings.
+		 * @param string $id_base  The widget ID.
+		 *
+		 * @var string
+		 */
+		$title = ! empty( $instance['title'] ) ? (string) apply_filters( 'widget_title', $instance['title'], $instance, $this->id_base ) : null;
+
+		/**
+		 * Filter the widget text.
+		 *
+		 * @link  https://developer.wordpress.org/reference/hooks/widget_text/
+		 * @since NEXT
+		 *
+		 * @param array                   $instance    Array of settings for the current widget.
+		 * @param Primer_Hero_Text_Widget $this        Current Hero Text widget instance.
+		 *
+		 * @var string
+		 */
+		$text = ! empty( $instance['text'] ) ? (string) apply_filters( 'widget_text', $instance['text'], $instance, $this ) : null;
+
+		$button_text = ! empty( $instance['button_text'] ) ? $instance['button_text'] : null;
+		$button_link = ! empty( $instance['button_link'] ) ? $instance['button_link'] : null;
+
+		// The `button_link` can be empty.
+		if ( ! $title && ! $text && ! $button_text ) {
+
+			return;
+
+		}
+
+		echo $args['before_widget']; // xss ok.
+
+		?>
+		<div class="textwidget primer-widgets primer-hero-text-widget">
+
+			<?php if ( $title ) : ?>
+
+				<?php echo $args['before_title'] . esc_html( $title ) . $args['after_title']; // xss ok. ?>
+
+			<?php endif; ?>
+
+			<?php if ( $text ) : ?>
+
+				<?php echo wp_kses_post( wpautop( $text ) ); ?>
+
+			<?php endif; ?>
+
+			<?php if ( $button_text ) : ?>
+
+				<p><a href="<?php echo esc_url( $button_link ); ?>" class="button"><?php echo esc_html( $button_text ); ?></a></p>
+
+			<?php endif; ?>
+
+		</div>
+		<?php
+
+		echo $args['after_widget']; // xss ok.
+
+	}
+
+	/**
+	 * Display the widget form fields.
+	 *
+	 * @since NEXT
+	 *
+	 * @param array $instance The widget instance values.
+	 */
+	public function form( $instance ) {
+
+		$title       = isset( $instance['title'] )       ? $instance['title']       : null;
+		$text        = isset( $instance['text'] )        ? $instance['text']        : null;
+		$button_text = isset( $instance['button_text'] ) ? $instance['button_text'] : null;
+		$button_link = isset( $instance['button_link'] ) ? $instance['button_link'] : null;
+
+		?>
+		<script>
```

Comment:
@fjarrett We can removed that since we don't have drag and drop anymore I think.